sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -242,6 +255,18 @@ private IEnumerable<Instruction> BuildNewInstance(ExpressionSyntax expressionSyn
             return new[] { newObjectInstruction };
         }
 
+        private IEnumerable<Instruction> BuildNewArrayInstance(ExpressionSyntax syntaxNode,
+            IArrayTypeSymbol arrayTypeSymbol,
+            InitializerExpressionSyntax arrayInitializationNode)
+        {
+            var newInstructions = BuildNewInstance(syntaxNode, arrayTypeSymbol);
+
+            var callTarget = expressionService.GetOrDefault(syntaxNode);
+
+            return newInstructions.
```

Comment:
[cosmetic] With the current style the `.` is always put on the new line.

Reply:
[minor] You could merge the lines like:

Reply:
I know, but I'm not really a fan of putting multiple nested methods calls on one line. Maybe it's just what I'm used to, but I find it harder to debug and to read.